java/configuration/src/main/java/sleeper/configuration/properties/table/TableProperty.java

```diff
@@ -380,6 +381,11 @@ public interface TableProperty extends SleeperProperty {
                     "If this minimum has not been reached, bulk import jobs will refuse to start")
             .defaultProperty(DEFAULT_BULK_IMPORT_MIN_LEAF_PARTITION_COUNT)
             .propertyGroup(TablePropertyGroup.BULK_IMPORT).build();
+    TableProperty BULK_IMPORT_FILES_COMMIT_ASYNC = Index.propertyBuilder("sleeper.table.bulk.import.job.files.commit.async")
+            .defaultProperty(DEFAULT_BULK_IMPORT_FILES_COMMIT_ASYNC)
+            .description("If true, ingest tasks will add files via requests sent to the state store committer lambda " +
+                    "asynchronously. If false, ingest tasks will commit new files synchronously.")
```

Comment:
This description needs updating to reference bulk import instead of ingest tasks

Reply:
Done, thanks